pkgs/dkg/drand.go

```diff
@@ -512,3 +499,15 @@ func (o *LocalOwner) broadcastError(err error) {
 	o.Broadcast(errMsg)
 	close(o.done)
 }
+
+func (o *LocalOwner) VerifyInitiatorMessage(msg []byte, sig []byte) error {
+	pubKey, err := crypto.EncodePublicKey(o.InitiatorPublicKey)
+	if err != nil {
+		return err
+	}
+	if err := crypto.VerifyRSA(o.InitiatorPublicKey, msg, sig); err != nil {
+		return fmt.Errorf("failed to verify a message from initiator: %x", pubKey)
+	}
+	o.Logger.Info("Successfully verified initiator message signature", zap.Uint64("from", o.ID))
+	return nil
```

Comment:
`VerifyInitiatorMessage` is used only in `state` hence it doesn't make sense to have it here, not the function and not the public key itself.

Reply:
I ill move it to instWrapper, as you suggested.